contracts/test/LON/Burn.t.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "contracts-test/LON/Setup.t.sol";
+
+contract TestLONBurn is TestLON {
+    // include Snapshot struct from BalanceSnapshot
```

Comment:
I think this comment could be removed, also for other places.

Reply:
Sure! I will remove these no longer used comments in the next commit.